src/config.rs

```diff
@@ -38,11 +38,14 @@ pub struct Config {
     // Bitwarden connection config
     bitwarden_url: String,
     bitwarden_admin_token: String,
+    bitwarden_root_cert: Option<String>,
```

Comment:
Can we rename this to `bitwarden_root_cert_file` or `bitwarden_root_cert_path`? That way it's clear that it's not the inline contents of the cert but rather a path to a file.

This feedback would include renaming references as well.

Reply:
Agreed, renamed to `bitwarden_root_cert_file`